### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by 4ndrewJ**
Please check formatting, naming and logic.
Thanks :)

@Z4MUS

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+"""
+CP1404/CP5632 Practical
+Get and store name and email
+"""
+
+
+def main():
+    """
+    Gets user input of names and emails
```
Comment (Z4MUS) [inline prac_05/emails.py]: Make sure to write your docstrings in the imperative voice.

So for "gets user input of names and emails" just change it to "get user input of names and emails".

see week 3 - functions slides for more details

File: prac_05/emails.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            name = input('Name: ')
+        name_to_email[name] = email
+        email = input('Email: ')
+    print()
+    display_names_and_emails(name_to_email)
+
+
+def display_names_and_emails(name_to_email):
+    """
+    Displays the names and emails in the dictionary name_to_email
```
Comment (Z4MUS) [inline prac_05/emails.py]: imperative voice (see comment on line 9)

File: prac_05/emails.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    """
+    for name_and_email in name_to_email.items():
+        print('{} ({})'.format(*name_and_email))
+
+
+def find_name_from_email(email):
+    """
+    Finds the section of email commonly used for names
```
Comment (Z4MUS) [inline prac_05/emails.py]: imperative voice (see comment on line 9)

**Comment by 4ndrewJ** [comment]
Thanks for the code review, I've made the requested changes and will now merge this branch.

**Pull request merged**